### PR TITLE
fix(components/atom/actionButton/demo): Updated demo from playground to js file

### LINKED ISSUE #1716: Documentation: actionButton - Missing examples of some props from the API tab
**Describe the problem**
There are some items in the API tab without an example in the demo

**Steps to reproduce**
1. Go to [actionButton](https://sui-components.vercel.app/workbench/atom/actionButton/demo)
2. Click on the API Tab

**Expected behaviour**
Every prop from the tab should have a demo

### PR DESCRIPTION
**Pull request by stefi23**
## atom/actionButton
**TASK**: #1716

### Types of changes

- [x] Documentation Only
- [x] Refactor

### Description, Motivation and Context
- Refactor demo files
- Add examples for missing API to demo

### Screenshots - Animations
Screenshot 2021-10-18 at 12-10-35 SUI Studio

### CODE REVIEW

**Comment by andresin87** [comment]
remove the playground file

**Comment by andresin87** [comment]
@turolopezsanabria can you check the deployment?

**Comment by turolopezsanabria** [comment]
👏🏼 thnx!

**Review by turolopezsanabria — APPROVED** [review]

File: components/atom/actionButton/src/ActionButton.js [hunk 1]
```diff
@@ -1,72 +1,145 @@
...
+AtomActionButton.displayName = 'AtomActionButton'
+
+AtomActionButton.propTypes = {
   /**
    * Content to be included in the button
    */
   children: PropTypes.node,
   /**
-   * Disable: faded with no interaction.
+   * Classes to add to button
```
Comment (turolopezsanabria) [inline components/atom/actionButton/src/ActionButton.js]:
```suggestion
   * Classes to add to button (DEPRECATED)
```

**Review by andresin87 — APPROVED** [review]

**Pull request merged**